redaxo/src/addons/metainfo/README.de.md

```diff
@@ -0,0 +1,8 @@
+# Meta Infos-AddOn
```

Comment:
Bin mir über die richtige Schreibweise nicht sicher. Aber so ist das glaube ich falsch, mit nur einem Bindestrich.
Denke es muss im Deutschen eher `Meta-Infos-AddOn` heißen, oder `Metainfos-AddOn`.
Bin mir auch unsicher, ob man es überhaupt in den Plural setzen sollte. Nicht eher `Meta-Info-AddOn`?

Ich denke, dass auch der Menüpunk im Backend mit `Meta Infos` falsch ist. Auch da gehört ein Bindestrich hin, oder zusammengeschrieben.

Reply:
Hab's noch mal umgeschrieben